verifier.py

```diff
@@ -160,7 +159,7 @@ def harvest_pages(self, json_list, url_end, page_class):
         :return: Null, but self.pages is populated.
         """
         for url in json_list:
-            if url.endswith("fail") == False and url.endswith(url_end):
+            if not url.endswith("fail"):
```

Comment:
Just wondering if there's a Pythonic/programming reason: why is it nested instead of and-ed now?